lua/autorun/client/cl_contact_forms.lua

```diff
@@ -810,4 +810,31 @@ net.Receive( "CFC_ContactForms_FailureAlert", function()
     )
 end )
 
+net.Receive("cfc_contact_forms_alert", function()
+    --local ply = net.ReadEntity() -- reads the reporter's player entity
+    local data = net.ReadTable()
+    local reporter = player.GetBySteamID(data["steam_id"])
+    local reported = player.GetBySteamID(data["reported_steam_id"])
+
+    local reporterName = reporter:GetName() -- reporter's name
+    local reporterRankColor = team.GetColor(reporter:Team()) -- color of reporter's rank
+
+    local reportedName = reported:GetName() -- reported player name
+    local reportedRankColor = team.GetColor(reported:Team()) -- color of reported player rank
+
+    chat.AddText(
+        reporterRankColor, reporterName,
+        YELLOW, " has submitted a report against ",
+        reportedRankColor, reportedName
+    ) -- message via chat
+    LocalPlayer():PrintMessage( 4, reporterName .. " made a report!" ) -- message via HUD
```

Comment:
Could be a bit generic, nitpick tho.

Reply:
Made both the same and specified "form report".